Configure

```diff
@@ -914,6 +917,16 @@ while (@argvcopy)
                             push @seed_sources, $x;
                             }
                         }
+                elsif (/^--fips-key=(.*)$/)
+                        {
+                        $user{FIPSKEY}=lc($1);
+                        die "Non-hex character in FIPS key\n"
+                           if $user{FIPSKEY} =~ /[^a-f0-9]/;
```

Comment:
Maybe allow uppercase hex letters, too?

Reply:
line 922 converts to lowercase.

Reply:
Ah, ok, I overlooked that.  Two letters, very unix-ish.